core/tracepoint/enable_spec.rb

```diff
@@ -0,0 +1,50 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.0' do
+  describe 'TracePoint#enable' do
```

Comment:
While this tests `enable` with no block, we're missing specs with `enable { 'with_block' }`